### PR TITLE
Expand the guide

### PR DESCRIPTION
**Pull request by pivovarit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,13 +72,13 @@ $ docker logs hazelcast-quarkus1 -f
...
 ```
 $ curl -X POST "localhost:8081/hazelcast/put?key=key_1&value=value_1";echo;
 {"containerName":"hazelcast-quarkus_2","value":"value_1"}
 ```
 
-Get the value from microservice running on 8080 and verify that it is the same value as put operation:
+Get the value from application running on 8080 and verify that it's the same value as put operation:
 ```
 $ curl "localhost:8080/hazelcast/get?key=key_1";echo;
 {"containerName":"hazelcast-quarkus_1","value":"value_1"}
```
Comment (mesutcelik) [inline README.md:84]: Do you think we should rename `hazelcast-quarkus_1` too? quarkus_replica_1 or microservice_replica_1 could be a better name.

File: pom.xml [hunk 2]
```diff
@@ -1,119 +1,127 @@
...
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-bom</artifactId>
-        <version>${quarkus.version}</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast-all</artifactId>
+            <version>3.12.4</version>
```
Comment (mesutcelik) [inline pom.xml:36]: that needs to be updated too.

**Review by mesutcelik — APPROVED** [review]
👍 I have two minor comments

File: pom.xml (hunk 2 above)
Comment (pivovarit) [reply pom.xml:36]: Yes, I will update in a separate PR (+ it will be guarded by Dependabot in the future)

File: README.md (hunk 1 above)
Comment (pivovarit) [reply README.md:84]: I don't really have a preference, I will update in the next PR

**Pull request merged**